### DISCUSSION TITLE
When i update my firmware,Is there anyway to backup my model first?

### DISCUSSION THREAD

**Discussion by zhangbeiforever** [OpenI6X/opentx#345] [Q&A]
Hi~~~
When I update my firmware,my model was lost. Is there anyway to backup my model first?
Thank you for answer.

**Comment by psicho100pro** [comment]
Only for some specific transitions from version to version this does not work.
https://www.youtube.com/watch?v=Z5zY_yA0avw

**Reply by zhangbeiforever** [reply]
Thank you
